### PR TITLE
docs: sd-container docs [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/styles/container/container.stories.ts [hunk 1]
```diff
@@ -7,11 +7,12 @@ const { overrideArgs } = storybookHelpers('sd-container');
 const { generateTemplate } = storybookTemplate('sd-container');
 
 /**
- * Container lets users delimit and highlight a piece of content. The user has no interaction with it, it is merely a visual element that influences the flow of the page.
+ * Allows users delimit and highlight a piece of content. The user has no interaction with it, it is merely a visual element that influences the flow of the page.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/container/container.stories.ts:10]: Allows users to delimit and highlight a piece of content that influences the flow of the page. The user has no interaction with it.

File: packages/components/src/styles/container/container.stories.ts [hunk 2]
```diff
@@ -52,178 +49,112 @@ export const Default = {
 };
 
 /**
- * Use the 5 color variants for alternate container experiences.
+ * Offers 5 color variants for alternate experiences:
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/container/container.stories.ts:52]: Offers “color” variants for alternate experiences:

File: packages/components/src/styles/container/container.stories.ts [hunk 3]
```diff
@@ -52,178 +49,112 @@ export const Default = {
...
+    </div>`
 };
 
 /**
- * You can add a triangle indentation to the container using the `--triangle-` class appended with one of the following positions 'top', 'right', 'bottom', 'left' (e.g. `sd-container--triangle-top`).
+ * You can add a triangle indentation to the container using the `triangle-class` appended with one of the following positions 'top', 'right', 'bottom', 'left' (e.g. sd-container--triangle-top).
+ *
+ * - A triangle can be shown to draw attention.
+ * - Triangle position can be `top`, `bottom`, `left` or `right`.
+ * - Default background option is white and can be overridden if desired.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/container/container.stories.ts:113]: For the preview of Triangle position I think we should use the border neutral 400. What do you think @coraliefeil ?

File: packages/components/src/styles/container/container.stories.ts [hunk 4]
```diff
@@ -52,178 +49,112 @@ export const Default = {
 };
 
 /**
- * Use the 5 color variants for alternate container experiences.
+ * Offers 5 color variants for alternate experiences:
+ *- `neutral-100 `
+ *- `primary-100`
+ *- `primary-500`
```
Comment (coraliefeil) [inline packages/components/src/styles/container/container.stories.ts:55]: *- `primary`

File: packages/components/src/styles/container/container.stories.ts (hunk 1 above)
Comment (coraliefeil) [reply packages/components/src/styles/container/container.stories.ts:10]: Let’s shorten it to:

Used to enclose and highlight specific parts of content on a page.

File: packages/components/src/styles/container/container.stories.ts (hunk 2 above)
Comment (coraliefeil) [reply packages/components/src/styles/container/container.stories.ts:52]: _For consistency use also here:_

Use the variant “classes” for alternative appearances:

**Comment by coraliefeil** [comment]
See comments. Reassign when adjusted.

**Comment by yoezlem** [comment]
Same Issue like here: https://github.com/solid-design-system/solid/pull/1286#discussion_r1721217240
Story isn't reactive anymore.

File: packages/components/src/styles/container/container.stories.ts [hunk 5]
```diff
@@ -20,210 +19,120 @@ export default {
...
 };
 
 /**
- * Use the 5 color variants for alternate container experiences.
+ * Use the variant “classes” for alternative appearances:
+ *- `neutral-100` (default): use the class `sd-container--variant-neutral-100`
+ *- `primary-100`: use the class `sd-container--variant-primary-100`
+ *- `primary`: use the class `sd-container--variant-primary`
+ *- `border-neutral-400`: use the class `sd-container--border-neutral-400`
+ *- `white`: use the class `sd-container--variant-white`:`
```
Comment (yoezlem) [inline packages/components/src/styles/container/container.stories.ts:35]:
```suggestion
 *- `white`: use the class `sd-container--variant-white`:
```

File: packages/components/src/templates/container.stories.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/ffdz9zO1CISlr8aHCZ7Bzp/Container?type=design&node-id=0-1&mode=design&t=pa9I1YKCYZQxj9Ob-0'
+    }
+  }
+};
+
+export const Default = {
```
Comment (coraliefeil) [inline packages/components/src/templates/container.stories.ts:17]: Hide Default

File: packages/components/src/templates/container.stories.ts [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+};
+/**
+ * You can set the color of the triangle cut-out using the `--triangle-background` CSS property. CSS variables can be set either with an inline style: `style="--triangle-background: rgb(var(--sd-color-primary-600, 0 53 142) / 1);"` or a custom class:
+ * `.custom-sd-container {
+    --triangle-background: rgb(var(--sd-color-primary-600, 0 53 142) / 1);
+  }`
+ */
+
+export const TriangleColor = {
+  name: 'Triangle Color',
```
Comment (coraliefeil) [inline packages/components/src/templates/container.stories.ts:31]: Container with Triangle Color

File: packages/components/src/styles/container/container.stories.ts [hunk 8]
```diff
@@ -7,223 +7,157 @@ const { overrideArgs } = storybookHelpers('sd-container');
...
-          alternate: 'y',
-          colors: ['tranparent', 'tranparent', 'tranparent', 'transparent', 'rgb(var(--sd-color-primary, 0 53 142))']
-        }
-      },
-      args
-    })}`;
-  }
+  render: () =>
+    html`<div class="grid grid-cols-2 gap-4">
+      <div class="sd-container default">
```
Comment (mariohamann) [inline packages/components/src/styles/container/container.stories.ts:50]: What does "default" do here?

File: packages/components/src/styles/container/container.stories.ts [hunk 9]
```diff
@@ -7,223 +7,157 @@ const { overrideArgs } = storybookHelpers('sd-container');
...
-  }
+  render: () =>
+    html`<div class="sd-container sd-container--padding-sm">
+      <div class="slot slot--border slot--text h-12">Default slot</div>
+    </div>`
 };
 
 /**
  * You can override the default padding values by setting your desired values as a style.
  */
```
Comment (mariohamann) [inline packages/components/src/styles/container/container.stories.ts:85]:
```suggestion
/**
 * You can override the default padding values via CSS.
 */
```

**Review by mariohamann — APPROVED** [review]
Fine besides the two comments I wrote. :)

**Pull request merged**